CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 This format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.19.3](https://github.com/flipt-io/flipt/releases/tag/v1.19.3) - 2023-22-15
```

Comment:
this date looks wrong, think it should be `2023-03-22`

Reply:
oops thank you. I cant time.